.storybook/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@docs/storybook",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:ts": "tsc --noEmit",
+    "dev": "storybook dev -c ./ -p 6006",
+    "preview": "NODE_ENV=production pnpm dev --docs",
+    "build": "storybook build -c ./ -o ./public",
+    "test": "test-storybook -c ./ --skipTags='skip-test' --url http://127.0.0.1:6006",
+    "clean": "rimraf ./public && rimraf ./node_modules"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.9.0",
```

Comment:
Does this need a downgrade or is it actually separate from `eslint`? 🤔

Reply:
Nope, can be removed since all linting is done from root